src/read/mod.rs

```diff
@@ -449,6 +449,34 @@ impl<'data> Export<'data> {
     }
 }
 
+/// PDB Information
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub struct CodeView<'data> {
+    guid: &'data [u8; 16],
```

Comment:
Let's change it from a reference to a value too, the size of this struct isn't significant.

Reply:
I can definitely do that, but I'm curious what benefit this brings? The `path` value will still need to be a reference with a lifetime of `'data` unless we want to copy that as well meaning this struct will still need to have the `'data` lifetime.

Reply:
For consistency with `mach_uuid`, and for the incredibly minor benefit that the user doesn't need to dereference it. I think UUIDs are like a primitive type that you normally pass around by value, not by reference.

Reply:
Makes sense, thanks for the explanation!